### ISSUE TITLE
java.lang.OutOfMemoryError from too many errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [useataaaaa/acra#50]
```
What steps will reproduce the problem?
1. Have way to many errors
2. It tries to send them
3. It runs out of memory....

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?
3.1.2 Android 2.2

Please provide any additional information below.

java.lang.OutOfMemoryError
    at java.lang.AbstractStringBuilder.enlargeBuffer(AbstractStringBuilder.java:97)
    at java.lang.AbstractStringBuilder.append0(AbstractStringBuilder.java:157)
    at java.lang.StringBuilder.append(StringBuilder.java:217)
    at org.acra.HttpUtils.doPost(HttpUtils.java:79)
    at org.acra.ErrorReporter.sendCrashReport(ErrorReporter.java:655)
    at org.acra.ErrorReporter.checkAndSendReports(ErrorReporter.java:759)
    at org.acra.ErrorReporter$ReportsSenderWorker.run(ErrorReporter.java:109)
```

Original issue reported on code.google.com by `federico...@gmail.com` on 29 Apr 2011 at 1:44

**Comment by GoogleCodeExporter** [comment]
```
I suspect this was fixed on 23-May-2011 and released in in 4.0.1b.
I think the OutOfMemory is to do with the Request size which deaults to 
megabytes on some devices. As of 4.0.1b ACRA defaults the Request size to 8K.
```

Original comment by `william....@gmail.com` on 16 Jul 2011 at 6:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.gaudin` on 2 Jan 2012 at 9:15
- Changed state: **Verified**
